### PR TITLE
Remove location add total and address

### PR DESCRIPTION
**Pull request by Gazwai**
Added a hidden form on the new form for the total to be submitted, and added address and postcode. The removed location from the new dress form.

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -2,10 +2,8 @@ class Booking < ApplicationRecord
   belongs_to :dress
   belongs_to :user
 
-  validates :dress, presence: true
-  validates :user, presence: true
-  validates :starting_date, presence: true
-  validates :ending_date, presence: true
+  validates :dress, :dress, :starting_date, :ending_date, :address, :post_code, :total_price, presence: true
```
Comment (angrysun) [inline app/models/booking.rb]: do we need the two :dress ? @Gazwai

**Comment by angrysun** [comment]
Looks good! Nice idea with the placeholder.

File: app/models/booking.rb (hunk 1 above)
Comment (Gazwai) [reply app/models/booking.rb]: Updated the validates to only one ":dress". Nice catch man

**Comment by Gazwai** [comment]
Screenshot 2022-02-02 at 13 18 11

**Pull request merged**